### ISSUE TITLE
Accessibility: Cannot focus out of the integrated terminal

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/vscode#6731]
Testing #6654 

* click into the integrated terminal
* try to `Tab` out, does not work
* in other places (editor) we use `ctrl+m` as a toggle between the widget trapping focus or not

**Comment by Tyriar** [comment]
Need https://github.com/sourcelair/xterm.js/issues/118 to be pushed for this.
